vercel_blob/blob_store.py

```
@@ -182,12 +219,14 @@ def put(path: str, data: bytes, options: dict = {}) -> dict:
         'PUT',
         headers=headers,
         data=data,
+        timeout=timeout,
+        verbose=verbose,
     )
 
     return _response_handler(resp)
 
 
-def head(url: str, options: dict = {}) -> dict:
+def head(url: str, options: dict = {}, timeout: int = 10) -> dict:
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)